wiki/BanchoBot/id.md

```diff
@@ -137,13 +137,13 @@ Terdapat 7 status yang dapat ditampilkan oleh perintah `!stats`: Editing, Idle,
 
 ### Report
 
-*Untuk mengetahui informasi tentang apa yang layak untuk dilaporkan, silahkan kunjungi artikel [Melaporkan Perilaku Buruk](/wiki/Reporting_Bad_Behaviour).*
+*Untuk mengetahui informasi tentang apa yang layak untuk dilaporkan, silahkan kunjungi artikel [Melaporkan perilaku buruk](/wiki/Reporting_Bad_Behaviour).*
 
 ```
 !report <nama pemain> <alasan>
 ```
 
-Perintah `!report` berfungsi memberi tahu [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) mengenai perilaku pemain yang tidak pantas. Jika nama pemain yang hendak dilaporkan mengandung karakter spasi, gantilah dengan garis bawah (cth. `really cool username` menjadi `really_cool_username`). Untuk melaporkan seorang moderator, silakan hubungi [account support team](/wiki/People/The_Team/Account_support_team#support@ppy.sh). Berikut contoh hasil dari perintah tersebut:
+Perintah `!report` berfungsi memberi tahu [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) mengenai perilaku pemain yang tidak pantas. Jika nama pemain yang hendak dilaporkan mengandung karakter spasi, gantilah dengan garis bawah (contoh `really cool username` menjadi `really_cool_username`). Untuk melaporkan seorang moderator, silakan hubungi [Account support team](/wiki/People/The_Team/Account_support_team#support@ppy.sh). Berikut contoh hasil dari perintah tersebut:
```

Comment:
this is not capitalised in the english article

```suggestion
Perintah `!report` berfungsi memberi tahu [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) mengenai perilaku pemain yang tidak pantas. Jika nama pemain yang hendak dilaporkan mengandung karakter spasi, gantilah dengan garis bawah (contoh `really cool username` menjadi `really_cool_username`). Untuk melaporkan seorang moderator, silakan hubungi [account support team](/wiki/People/The_Team/Account_support_team#support@ppy.sh). Berikut contoh hasil dari perintah tersebut:
```

Reply:
Ah this team doesn't follow naming format like GMT or NAT?

Reply:
nope. if you're referring to the team, it's lowercase, but if you're referring to the article itself, then it's the same as the article title

Reply:
but Support Team would be capitalised (see [`wiki/People/The_Team`](https://github.com/ppy/osu-wiki/tree/master/wiki/People/The_Team))